### ISSUE TITLE
Sidekiq needs to be manually started

### ISSUE THREAD

**Issue by orangewolf** [scientist-softserv/louisville-hyku#69]
Sidekiq needs to be manually started in web_1 Docker container.

**Comment by orangewolf** [comment]
Related Merge Requests
---
[121-remove-auto-expiry-jobs](https://gitlab.com/notch8/louisville-hyku/-/merge_requests/128)

**Comment by orangewolf** [comment]
Randy June 2022
---
Per [@alishaevn](https://github.com/alishaevn) happens in dev env only

**Comment by orangewolf** [comment]
Randy June 2022
---
I'm going to reopen this one. Sidekiq needs to be manually started even if it's currently running as a service. If it's not manually started, documents become stuck in the busy queue and will never be processed. This occurs immediately after bringing up a new instance up (EmbargoAutoExpiryJob & LeaseAutoExpiryJob remain stuck under busy) and also occurs when adding new records.

**Comment by orangewolf** [comment]
Alisha June 2022
---
@rtkuehn those 2 jobs holding up things is also an issue I've experienced in local dev. typically I stop sidekiq and delete those jobs, then restart sidekiq. which allows the rest of the jobs to continue.

looking into whether there is an actual solution though.[@rtkuehn](https://github.com/rtkuehn) those 2 jobs holding up things is also an issue I've experienced in local dev. typically I stop sidekiq and delete those jobs, then restart sidekiq. which allows the rest of the jobs to continue.
looking into whether there is an actual solution though.
